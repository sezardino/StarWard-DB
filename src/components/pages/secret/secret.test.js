import React from 'react';
import {shallow} from 'enzyme';

import SecretPage from './secret';

describe('Component SecretPage', () => {
	const component = shallow(<SecretPage isLoggedIn={false} />);
	it('Component rendered correctly when isLoggedIn is false', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
