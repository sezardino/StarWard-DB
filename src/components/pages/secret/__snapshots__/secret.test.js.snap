// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component SecretPage Component rendered correctly when isLoggedIn is false 1`] = `"<Redirect to=\\"/login\\" />"`;
